yarnpkg/berry
# feat(plugin-pack): ignore Sapling from `yarn pack`

Summary:
This change adds `.sl` to the list of directories ignored by `yarn pack`.

Test Plan:
- Clone a Sapling repo
- Put some JS files in the `.sl` directory (e.g. `touch .sl/package.json`)
- Run `yarn pack`
- Verify that the `.sl` directory is not included in the tarball
